src/Main.cc

```diff
@@ -522,29 +531,46 @@ int main(int argc, const char* argv[]) {
         int port(3002);
         int thread_count = TBB_THREAD_COUNT;
         int omp_thread_count = OMP_THREAD_COUNT;
+        string frontend_folder;
+        string host;
+        bool no_http = false;
+        bool debug_no_auth = false;
+        bool no_browser = false;
+
         { // get values then let Input go out of scope
             casacore::Input inp;
             inp.version(VERSION_ID);
             inp.create("verbose", "False", "display verbose logging", "Bool");
             inp.create("perflog", "False", "display performance logging", "Bool");
+            inp.create("no_http", "False", "disable CARTA frontend HTTP server", "Bool");
+            inp.create("debug_no_auth", "False", "accept all incoming WebSocket connections (insecure, use with caution!)", "Bool");
+            inp.create("no_browser", "False", "prevent the frontend from automatically opening in the default browser on startup", "Bool");
+            inp.create("host", "", "only listen on the specified interface");
             inp.create("port", to_string(port), "set server port", "Int");
```

Comment:
Should the description be updated to clarify that the same port will be used by the backend and frontend?

Do we want to make these separately configurable? Is there a reason to do that?

Reply:
WebSocket connections are actually an HTTP connection that is upgraded to WebSocket via the [UPGRADE header](https://en.wikipedia.org/wiki/HTTP/1.1_Upgrade_header), so it's really the same server hosting both the HTTP connections to server the frontend _and_ the WebSocket connections to the backend. I don't think it's really possible to make those ports separately configurable unless we run two separate servers.

Reply:
I've added some additional text for the `port` argument